src/javascript/prettier.ts

```diff
@@ -0,0 +1,401 @@
+import { Component } from '../component';
+import { NodeProject } from '../javascript';
+import { JsonFile } from '../json';
+import { Project } from '../project';
+
+export interface PrettierConfig {
```

Comment:
All public APIs should have docstrings:

```suggestion
/**
 * Options for `Prettier`.
 */
export interface PrettierConfig {
```